code/modules/cooking_with_jane/porting_readme.txt

```diff
@@ -0,0 +1,22 @@
+Things to keep in mind when porting to eris:
+
+- Cooking has to be initialized with initialize_cooking_recipes() in /datum/global_init/New().
+It has to be ran AFTER initialize_chemical_reagents() and initialize_chemical_reactions()!
+
+- The CtrlShiftClickOn, CtrlShiftClick, CtrlClickOn and CtrlClick procs need to be updated to
+recieve the params value from the ClickOn function. This is needed for the stove and grill
+to function.
+
+- Any atom used as an ingredient in cooking needs a 'food_quality' variable.
+
+- Any atom produced by cooking needs a 'food_quality' variable AND a 'cooking_description_modifier' variable.
+
+- Modify slicable food to inherit the food quality of their parent.
+
+- The function call createCookingCatalogs() has to be added at the end of the
+/hook/startup/proc/createCatalogs() function.
```

Comment:
This should be documented in the CL

Reply:
Done.

Reply:
No, i mean the CtrlShiftClickOn, CtrlShiftClick, CtrlClickOn and CtrlClick changes are major changes and should be documented, also the fact that # exclusion was added to the sanitizeFileName.

Reply:
I believe I added them appropriately, but I didn't signify every proc individually, I also added the reason.